manifests/daemonset_openshift.yaml

```diff
@@ -0,0 +1,169 @@
+        securityContext:
+          privileged: true
```

Comment:
Would you remind me why the ds pod must be privileged ?

Reply:
Because the cni pod writes to /host/opt directory